mvg/plotting.py

```diff
@@ -210,6 +210,18 @@ def modes_over_time(
     else:
         data["Date"] = pd.to_datetime(data["timestamps"], unit=timeunit)
 
+    # Add an additional row with a future timestamp to ensure the mode
+    # for the last timestamp in the dataset is clearly visible
+    ts_end, ts_prev = data["timestamps"].iloc[[-1, -2]]
+    ts_next = (ts_end - ts_prev) + ts_end
+    data.loc[len(data)] = {
+        "timestamps": int(ts_next),
+        "labels": None,
```

Comment:
It is better to repeat the label from the last timestamp for this extra one so we don't need to drop the `None` in the end. Ideally this can be done for all other values expect `timestamp`
```suggestion
        "labels": data["labels"][-1],
```